modules/core/arrow-data/src/main/kotlin/arrow/data/Sum.kt

```diff
@@ -17,7 +17,7 @@ data class Sum<F, G, V>(
     object Right : Side()
   }
 
-  fun <A> coflatMap(CF: Comonad<F>, CG: Comonad<G>, f: (Sum<F, G, V>) -> A): Sum<F, G, A> = Sum(
+  fun <A> coflatmap(CF: Comonad<F>, CG: Comonad<G>, f: (Sum<F, G, V>) -> A): Sum<F, G, A> = Sum(
```

Comment:
It's coflatMap AFAIK.

Reply:
You changed it recently from `extend` to `coflatmap` and this was the only left to change. I'll change them all to `coflatMap`.